media-video/mjpg-streamer/mjpg-streamer-0_pre20250324-r1.ebuild

```
@@ -0,0 +1,99 @@
+# Copyright 1999-2025 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit systemd
+
+DESCRIPTION="MJPG-streamer takes JPGs from Linux-UVC compatible webcams"
+HOMEPAGE="https://github.com/jacksonliam/mjpg-streamer"
+EGIT_COMMIT="310b29f4a94c46652b20c4b7b6e5cf24e532af39"
+SRC_URI="https://github.com/jacksonliam/${PN}/archive/${EGIT_COMMIT}.tar.gz -> ${P}.tar.gz"
+S="${WORKDIR}/${PN}-${EGIT_COMMIT}/${PN}-experimental"
```

EGIT_COMMIT would be reserved for `git-r3.eclass`. This ebuild is not using `git-r3.eclass`. So just make it `COMMIT`.

```suggestion
COMMIT="310b29f4a94c46652b20c4b7b6e5cf24e532af39"
SRC_URI="https://github.com/jacksonliam/${PN}/archive/${COMMIT}.tar.gz -> ${P}.tar.gz"
S="${WORKDIR}/${PN}-${COMMIT}/${PN}-experimental"
```